test/validate/eia_test.py

```diff
@@ -45,13 +45,13 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_607_752, 1_607_752, 128_252),
+        ("bf_eia923", 1_607_692, 1_607_692, 128_252),
         ("bga_eia860", 142_391, 142_391, 142_391),
-        ("boil_eia860", 83_416, 83_416, 83_416),
-        ("frc_eia923", 650_671, 265_784, 25_443),
+        ("boil_eia860", 83_418, 83_418, 83_418),
+        ("frc_eia923", 656_388, 268_352, 25_443),
```

Comment:
Does a ~1% incrase in the number of rows in the FRC table seem correct? How many months of new data are we adding? Is it just a single quarter?  If so, then ~1% seems like the right order of magnitude, since we have ~16 years of this table.

Reply:
yup, it's about a quarter's worth of data